src/lib/components/DashPlayer.react.js

```diff
@@ -141,17 +142,23 @@ export default class DashPlayer extends Component {
                 height={height}
                 style={style}
                 playsinline={playsinline}
+                class={className}
```

Comment:
Does this work? https://github.com/CookPete/react-player#responsive-player shows a `className` prop, not `class` (though neither one shows up in their [props table](https://github.com/CookPete/react-player#props) 🙄 )

Reply:
Yes - here's an example of it working. And to address this [comment](https://github.com/plotly/dash-player/pull/47#discussion_r984652372), the `tester` class was just used to test out `className` I'll remove it in #48 

<img width="1294" alt="Screen Shot 2022-10-04 at 12 26 51 PM" src="https://user-images.githubusercontent.com/42301846/193873878-7cc76613-8fc0-471a-b436-223a95ce08ff.png">